src/components/Documentation/Layout/SidebarMenu/index.tsx

```diff
@@ -50,11 +50,31 @@ const SidebarMenuItem: React.FC<ISidebarMenuItemProps> = ({
   icon,
   type
 }) => {
-  const isActive = activePaths && includes(activePaths, path)
+  const [isActive, setIsActive] = useState(
+    activePaths && includes(activePaths, path)
+  )
```

Comment:
My main contention with this is that it steals the former `isActive`, which is useful as a value distinct from this feature. I would make this `useState` separate and called `isExpanded`/`setIsExpanded`.

Reply:
@rogermparent  What about the former isActive variable? It will not be used anywhere so should I just rename the variable names?

Reply:
Okay, it has been useful now based on the below comment, thank you. :D